extensions/amp-sidebar/1.0/sidebar-animations-hook.js

```diff
@@ -71,7 +73,10 @@ export function useSidebarAnimation(
     const backdropElement = backdropRef.current;
     // The component might start in a state where `side` is not known
     // This effect must be restarted when the `side` becomes known
-    if (!sidebarElement || !backdropElement || !side) {
+    // Must also check mounted as of #33244.  This is because previously
+    // sidebarElement and backdropElement would not be rendered when
+    // mounted is false.  This is no longer the case.
+    if (!mounted || !sidebarElement || !backdropElement || !side) {
       return;
```

Comment:
Added a check for `mounted` here.  Per the comment, we don't want animations running in an unmounted state.  Previously checking `sidebarElement` and `backdropElement` refs were enough since they were never rendered in `unmounted` state.